recipes-connectivity/opendds/opendds_3.24.bb

```diff
@@ -0,0 +1,41 @@
+# Specify SHA-1 for the release to avoid constantly checking the upstream repo.
+
+SRCREV = "4dd8c404253b8db95bfe3e98ea38a1b14ef228a8"
+DDS_SRC_BRANCH = "branch-DDS-3.24"
+SRC_URI = "git://github.com/objectcomputing/OpenDDS.git;protocol=https;branch=${DDS_SRC_BRANCH};name=opendds"
+
+require opendds.inc
+
+DOC_TAO2_VERSION = "6.5.19"
+DOC_TAO3_VERSION = "7.1.0"
```

Comment:
Is there a need to support both?  It would simplify maintenance to standardize on one.

Reply:
I will change it to support DOC_TAO2_VERSION which is 6.5.19 for OpenDDS 3.24.X